spec/features/check_answers_page_spec.rb

```diff
@@ -104,4 +104,19 @@
       expect(page).not_to have_content benefits_subsection_header
     end
   end
+
+  context "when I have a partner" do
+    before do
+      visit_check_answer_with_partner
+    end
+
+    scenario "I should see partner content" do
+      expect(page).to have_content "Partner's Employment"
```

Comment:
minor nitpick - should Employment be lower case to match the other titles on the page?